src/pixelator/pixeldataset/datastores.py

```diff
@@ -590,13 +608,18 @@ def write_dataframe(
                     partitioning_flavor="hive",
                     partitioning=partitioning,
                     use_threads=False,
+                    file_options=file_options,
                     existing_data_behavior="overwrite_or_ignore",
                 )
             return
 
         self._check_if_writeable(key)
         pq.write_table(
-            pa.Table.from_pandas(dataframe), where=key, filesystem=self._file_system
+            pa.Table.from_pandas(dataframe, preserve_index=False),
+            where=key,
+            filesystem=self._file_system,
+            compression=DEFAULT_COMPRESSION,
+            row_group_size=len(dataframe) or 1,
```

Comment:
MAybe it is not immediate on why is like this. I wonder if we should include a comment with your findings yesterday afternoon

Reply:
Good point! I will do that.